grizli/pipeline/auto_script.py

```diff
@@ -1389,20 +1398,21 @@ def parse_visits(field_root='', RAW_PATH='../RAW', use_visit=True, combine_same_
 
             visits.append(direct)
 
-        all_groups = utils.parse_grism_associations(visits)
-        #np.save('{0}_visits.npy'.format(field_root), 
-        #        [visits, all_groups, info])
+        all_groups = utils.parse_grism_associations(visits, info, isJWST=isJWST)
+
         write_visit_info(visits, all_groups, info, root=field_root, path='./')
                 
         return visits, all_groups, info
 
     visits, filters = utils.parse_flt_files(info=info, 
-                                  uniquename=True, get_footprint=True, 
+                                  uniquename=True, get_footprint=True,
                                   use_visit=use_visit, max_dt=max_dt, 
-                                  visit_split_shift=visit_split_shift)
+                                  visit_split_shift=visit_split_shift,
+                                  isJWST=isJWST)
+    
 
     # Don't run combine_minexp if have grism exposures
-    grisms = ['G141', 'G102', 'G800L', 'G280']
+    grisms = ['G141', 'G102', 'G800L', 'G280', 'GR150C', 'GR150R']
     has_grism = utils.column_string_operation(info['FILTER'], grisms,
```

Comment:
See above for `run_has_grism`

Reply:
Done here: ef053b3aed8f8e694f9e8488e76919c9f68b04ca